src/main/java/ch/njol/skript/util/ColorRGB.java

```diff
@@ -35,31 +20,96 @@ public class ColorRGB implements Color {
 	private static final Pattern RGB_PATTERN = Pattern.compile("(?>rgb|RGB) (\\d+), (\\d+), (\\d+)");
 
 	private org.bukkit.Color bukkit;
+
 	@Nullable
 	private DyeColor dye;
-	
+
+	/**
+	 * Subject to being private in the future. Use {@link #fromRGB(int, int, int)}
+	 * This is to keep inline with other color classes.
+	 */
+	@Deprecated
+	@ApiStatus.Internal
 	public ColorRGB(int red, int green, int blue) {
-		this.bukkit = org.bukkit.Color.fromRGB(
+		this(org.bukkit.Color.fromRGB(
 			Math2.fit(0, red, 255),
 			Math2.fit(0, green, 255),
-			Math2.fit(0, blue, 255));
+			Math2.fit(0, blue, 255)));
+	}
+
+	/**
+	 * Subject to being private in the future. Use {@link #fromBukkitColor(org.bukkit.Color)}
+	 * This is to keep inline with other color classes.
+	 */
+	@Deprecated
+	@ApiStatus.Internal
+	public ColorRGB(org.bukkit.Color bukkit) {
 		this.dye = DyeColor.getByColor(bukkit);
+		this.bukkit = bukkit;
+	}
+
+	private static final boolean HAS_ARGB = Skript.methodExists(org.bukkit.Color.class, "getAlpha");
```

Comment:
I would consider moving this to the top of the file. If not, a space below it would do some good (since it's touching the javadoc of the following method)